plugins/inventory/iocage.py

```diff
@@ -0,0 +1,263 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024 Vladimir Botka <vbotka@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: iocage
+    short_description: iocage inventory source
+    version_added: 10.2.0
+    author:
+        - Vladimir Botka (@vbotka)
+    requirements:
+        - iocage >= 1.8
+    description:
+        - Get inventory hosts from the iocage jail manager running on O(host).
+        - By default, O(host) is V(localhost). If O(host) is not V(localhost) it
+          is expected that the O(user) is able to connect to the O(host) with
+          SSH and execute the command C(iocage list).
```

Comment:
Technically, it is not the `O(user)` (remote user on the target iocage host) that must be able to connect. This would be more accurate as:
	"... expected that the user running Ansible on the controller is able to connect with SSH to the O(host) with O(user) without specifying password or key, and O(user) is able to execute the command C(iocage list)."

Or something along those lines - feel free to wordsmith it to your satisfaction.

Reply:
Thank you. I updated the description item

```
        - By default, O(host) is V(localhost). If O(host) is not V(localhost) it
          is expected that the user running Ansible on the controller can
          connect to the O(host) account O(user) with SSH non-interactively and
          execute the command C(iocage list).
```

Reply:
Thanks!